doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+    new_kernel=`ls /mnt/EFI/org.clearlinux/*sos* -tl | grep kernel | head -n1 | awk -F'/' '{print $5}'`
+    echo "Get current sos kernel from acrn.conf file: "
+    cur_kernel=`cat /mnt/loader/entries/acrn.conf | sed -n 2p | cut -d'/' -f4`
+
+    echo "Replace root uuid from acrn.conf"
+    sed -i "s/<UUID of rootfs partition>/`blkid -s PARTUUID -o value /dev/sda3`/g" /mnt/loader/entries/acrn.conf
+
+    if [[ "$cur_kernel" == "$new_kernel" ]]; then
+        echo "No need to replace kernel conf info"
+    else
+        echo "Replace with new sos kernel in acrn.conf"
+        sed -i "s/$cur_kernel/$new_kernel/g" /mnt/loader/entries/acrn.conf
+    fi
+
+    echo "Sos process done!"
+    echo " ***** Please reboot device to test on new sos. *****"
+else
+    echo -e "Fail to upgrade sos $target_clearlinux from mirror url."
+fi
+
+echo "Init swupd mirror & proxy"
+swupd mirror -u
+unset https_proxy
+umount /mnt
+}
+
+function upgrade_uos()
+{
+# Add proxy if you need to download uos img automatically.
+if [[ -n $proxy ]]; then
+    echo "Add proxy..."
+    export https_proxy=$proxy
+fi
+
+# Do upgrade uos process.
+if [[ $skip_download_uos == 1 ]]; then
+    uos_img_xz_path=$(find ~/ -name clear-$target_version-kvm.img.xz)
+    uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    if [[ ! -f $uos_img_xz_path ]] && [[ ! -f $uos_img_path ]]; then
+        echo "You should download uos clear-$target_version-kvm.img.xz file firstly." && exit
+    fi
+    if [[ -f $uos_img_xz_path ]]; then
+        echo "get uos xz file in: $uos_img_xz_path"
+        unxz $uos_img_xz_path
+        uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    fi
+    echo "get uos img file in: $uos_img_path"
+    if [[ $? -eq 0 ]]; then
+        losetup -f -P --show $uos_img_path
+    else
+        echo "fail to losetup uos img, please check uos img status" && exit 1
+    fi
+    mount /dev/loop0p3 /mnt
+    cp -r /usr/lib/modules/"`readlink /usr/lib/kernel/default-iot-lts2018 | awk -F '2018.' '{print $2}'`.iot-lts2018" /mnt/lib/modules
+    umount /mnt
+    sync
+else
+    cd ~
+    curl https://download.clearlinux.org/releases/$target_version/clear/clear-$target_version-kvm.img.xz -o clear-$target_version-kvm.img.xz
+    if [[ $? -eq 0 ]]; then
+        unxz clear-$target_version-kvm.img.xz
+    fi
+    if [[ $? -eq 0 ]]; then
+        losetup -f -P --show clear-$target_version-kvm.img
+    fi
+    mount /dev/loop0p3 /mnt
+    cp -r /usr/lib/modules/"`readlink /usr/lib/kernel/default-iot-lts2018 | awk -F '2018.' '{print $2}'`.iot-lts2018" /mnt/lib/modules
+    umount /mnt
+    sync
+fi
+uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+cp -r /usr/share/acrn/samples/nuc/launch_uos.sh ~/
+sed -i "s/\(virtio-blk.*\)\/home\/clear\/uos\/uos.img/\1$(echo $uos_img_path | sed "s/\//\\\\\//g")/" ~/launch_uos.sh
+echo "Upgrade uos done..."
+echo "Now you can run below command to start UOS..."
+echo "# cd ~/ && ./launch_uos.sh -V 1"
```

Comment:
We do not have this `-V` parameter in the launch scripts for EFI platforms

Reply:
Yes, it's reword to this 'cd ~/ && ./launch_uos.sh'